.travis.yml

```diff
@@ -0,0 +1,18 @@
+if: branch = master and type IN (pull_request)
+language: cpp
+services:
+  - docker
+env:
+before_install:
+  - docker pull fiware/orion-ci:rpm7
+install: true
+before_script: true
+script: true
+jobs:
+  include:
```

Comment:
As additional stages, I'd suggest to add:

* File compliance: corresponding to `make files_compliance`
* Style: corresponding to `make style`
* Payloads: corresponding to `make payload_check`

They have to be done with the repository after cloning and before starting building (due to the building process created additional directories, as BUILD_DEBUG, BUILD_RELEASE, etc., that the above checks are not able to manage properly and could produce false positives).

The time they take is negligible  (compared with the next ones for unit tests and functional tests) so they would be good additions to the checks done by travis.

Reply:
Ok, will add

Reply:
make style returns a lot of errors:
0043 errors of category 'Lines should be <= 120 characters long'
0004 errors of category 'Line ends in whitespace'
0091 errors of category 'Use int16/int64/etc'

what should i do with it?

Reply:
I think that master is currently style-broken due to a recent PR. Has been this PR branch updated with master recently?

By the moment don't pay too much attention to that... it will be eventually fixed in master (I'll tell you on this comment) while this PR gets completed.

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/e9647f964cd31c64d32f84ab6877d03921dcc09b